packages/utils/src/account.ts

```diff
@@ -0,0 +1,67 @@
+import * as bip32 from 'bip32'
+import * as bip39 from 'bip39'
+export const CELO_DERIVATION_PATH_BASE = "m/44'/52752'/0'/0"
+
+export enum MnemonicStrength {
+  s128_12words = 128,
+  s256_24words = 256,
+}
+
+export enum MnemonicLanguages {
+  chinese_simplified,
+  chinese_traditional,
+  english,
+  french,
+  italian,
+  japanese,
+  korean,
+  spanish,
+}
+
+export function generateMnemonic(
```

Comment:
Please package these functions up into a containing object and export it in index.ts (see utils/src/phoneNumbers.ts for an example) so it can be more easily consumed by non-TS consumers as well.